src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.js

```diff
@@ -43,9 +45,13 @@ const EditButton = styled(Button)`
   padding: ${themeSpacing(0, 1.5)};
 `;
 
+const getDirectingDepartmentValue = value => (value?.length === 1 && value[0].code === 'ASC' ? 'ASC' : 'null');
```

Comment:
`ASC` is an hard-coded constant we use it to determine the id the back-end is expecting. This is the way the directing departments are defined in the back-end and it is agreed to make this conversion here. Not happy with this either but I have no choice.

Reply:
*cough* Graphql *cough* Typescript *cough* 😇

Reply:
I don't understand. `ASC` is the code of a specific department, isn't it? This will not work then with other municipalities that don't have that department.